src/lib/config/types.ts

```diff
@@ -69,6 +70,8 @@ export interface Keys {
 export interface Config {
   key: string;
   chainId: Network | 12345 | 17;
+  layerZeroChainId?: number;
+  veBalSync?: boolean;
```

Comment:
Can you name this `supportsVeBalSync` and move to below `supportsElementPools` to be consistent with the others